develop.sh

```diff
@@ -158,18 +158,18 @@ ci_docker_login)
     ;;
 dev_aloe)
     shift
-    dev_aloe $@
+    dev_aloe aloe $@
     ;;
 aloe)
     shift
-    dev_aloe $@
+    dev_aloe aloe $@
     ;;
 reexport_test_zips)
     reexport_test_zips
     ;;
 prod_aloe)
     shift
-    prod_aloe $@
+    prod_aloe aloe $@
```

Comment:
Strictly speaking here is isn't needed. But downstream projects can pass different values for this arg. Changing it here so that there is one less point of difference.